src/components/scenes/Fio/FioRequestConfirmationScene.tsx

```diff
@@ -211,7 +211,8 @@ export class FioRequestConfirmationConnected extends React.Component<Props, Stat
       } catch (error: any) {
         this.setState({ loading: false })
         this.resetSlider()
-        showError(
+        showError(lstrings.fio_request_error_header)
+        showDevError(
```

Comment:
Since this is `onConfirm`, it's a user-triggered action and we definitely should be showing the error. The lines below can be simplified to `error.json?.fields?.[0]?.error ?? ''`, which seems like a reasonable thing to show the user as well. So, let's do that cleanup and drop the `showDevError`